### PR TITLE
Update apple.txt

### PR DESCRIPTION
**Pull request by parakramsinhrathod**
path 1 something check

### CODE REVIEW

File: apple.txt [hunk 1]
```diff
@@ -20,3 +20,4 @@ dooooooooooooooooooooooooo&@@oooooob
 jgs   `dooooooooobodoooooooob'
```
Comment (vishwajeetsinh7) [inline apple.txt:20]: ok it's fine

File: apple.txt [hunk 2]
```diff
@@ -20,3 +20,4 @@ dooooooooooooooooooooooooo&@@oooooob
...
        `doooooooob dooooooob'```
```
Comment (vishwajeetsinh7) [inline apple.txt:21]: okkkkk

**Review by vishwajeetsinh7 — COMMENTED** [review]
,finish review

**Comment by parakramsinhrathod** [comment]
thanks

File: apple.txt (hunk 2 above)
Comment (parakramsinhrathod) [reply apple.txt:21]: tthhank

**Comment by vishwajeetsinh7** [comment]
sad

File: apple.txt (hunk 2 above)
Comment (vishwajeetsinh7) [reply apple.txt:21]: s

**Review by vishwajeetsinh7 — APPROVED** [review]

**Comment by vishwajeetsinh7** [comment]
***@***.***
jnv panchmahal
***@***.***
tidal
***@***.***

**Pull request merged**